.github/workflows/nightly_tap.yml

```
@@ -126,11 +145,93 @@ jobs:
+  # the open-PR matrix above. Without this, a regression that lands on main
+  # is not exercised by the nightly until/unless it surfaces inside some
+  # open PR. Mirror the pull-and-test step list, but check out main and use
+  # a different container/artifact naming scheme so PR and main runs do not
+  # collide.
+  #
+  # TEMPORARILY DISABLED: 016_crash_recovery_progress and 017_zodan_3n_timeout
+  # are not yet skipped on main, so this job is guaranteed to fail until those
+  # are addressed (or this PR's disable lands on main). Flip `if: false` back
+  # to remove the gate when main is ready to be exercised.
+  test-main:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)